vspace/src/main/java/edu/asu/diging/vspace/core/factory/impl/ExhibitionFactory.java

```diff
@@ -9,10 +9,14 @@
 @Service
 public class ExhibitionFactory implements IExhibitionFactory {
 
-    /*
-     * (non-Javadoc)
+    public static final String EXH_PREVIEW = "EXH_PREVIEW_";
+
+    /**
+     * This method create an Exhibition with a previewId and its status has been set
+     * to active by default. This previewId is generated randomly every time an
+     * exhibition got created.
      * 
-     * @see edu.asu.diging.vspace.core.model.impl.Exhibition
+     * @return an Exhibition object after creation.
      */
     @Override
     public IExhibition createExhibition() {
```

Comment:
something was lost here at some point. The preview id should be created here not in the controller:
https://github.com/diging/virtual-spaces-2.0/pull/271/files/432773db40d0858ae7004d374c47f9275381d198#diff-6b78c69f4db9c9f70c0c8c8fe6ab411b02b32b1af791b5f7f72e6c97a5020afe